torch_frame/data/multi_tensor.py

```diff
@@ -325,6 +325,37 @@ def select(
     def _single_index_select(self, index: int, dim: int) -> _MultiTensor:
         raise NotImplementedError
 
+    def fillna(self, fill_value: Union[float, int]) -> None:
+        is_float = self.dtype.is_floating_point
+        if (is_float and isinstance(fill_value, int)
```

Comment:
I don't see an issue filling with an integer for floating-point values. Do we need this check? Why not let PyTorch handle any errors downstream.